cloudimg/aws.py

```diff
@@ -558,6 +566,13 @@ def publish(self, metadata):
         Returns:
             An EC2 Image
         """
+        def add_tags(tag_name, extra_kwargs):
```

Comment:
`tags_parameter_name` might be simpler to understand the code
I have a feeling `tag_name` might be confusing

Reply:
Fixed